app/src/org/odk/collect/android/views/ODKView.java

```diff
@@ -42,15 +41,12 @@
 
     // starter random number for view IDs
     private final static int VIEW_ID = 12345;  
-    
-    private static final String TAG = ODKView.class.getSimpleName();
 
     private final LinearLayout mView;
     private final LinearLayout.LayoutParams mLayout;
     private final ArrayList<QuestionWidget> widgets;
     private final ArrayList<View> dividers;
-    private ProgressBar mProgressBar;
```

Comment:
The progress bar now lives somewhere that is not the ODKView, so `updateProgressBar` was never called.